main/src/main/scala/org/clulab/odin/Mention.scala

```diff
@@ -102,6 +102,9 @@ trait Mention extends Equals with Ordered[Mention] with Serializable {
   /** returns all entities in mention */
   def entities: Option[Seq[String]] = sentenceObj.entities.map(_.slice(start, end))
 
+  /** returns the norm value of this mention, if available */
+  def norm: Option[String] = sentenceObj.norms.map(x => x(start))
```

Comment:
@MihaiSurdeanu maybe none of my business but why only the start and not the whole slice?